.github/workflows/chromaticqa.yml

```diff
@@ -0,0 +1,27 @@
+name: ChromaticQA
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  visual_regression:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        # Use 'head' instead of 'merge': https://docs.chromaticqa.com/setup_ci#pull-request-builds
+        # 'head' is the HEAD of the PR branch. 'merge' is the hidden merge commit created by github
+        ref: refs/pull/${{ github.event.pull_request.number}}/head
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+    - name: Install Packages
+      run: yarn install
+    - name: ChromaticQA
+      run: yarn chromatic
+      env:
+        CI: true
+        CHROMATIC_APP_CODE: dlpro96xybh
```

Comment:
We don't have a good way to hide these keys. I spoke with the ChromaticQA team and they recommended using the key in public until there is a better way to hide it. They can issue us a new app code on request.